docs/configuration.rst

```diff
@@ -860,6 +860,22 @@ Other
          These metrics track the memory management and concurrency of the python runtime. 
          Refer to the following `docs <https://docs.datadoghq.com/tracing/metrics/runtime_metrics/python/>` _ for more information.
 
+    DD_TRACE_EXPERIMENTAL_RUNTIME_ID_ENABLED:
+     type: Boolean
+     default: False
+
+     description: |
+         Enables support for experimental ddtrace configurations. This configuration supports a comma delimited list of ddtrace environment variables (only commas, underscores and alphanumeric characters are supported).
+         The supported configurations: ``DD_RUNTIME_METRICS_ENABLED``.
+
+    DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED:
+     type: String
+     default: ""
+
+     description: |
+         Adds support for tagging runtime metrics with the current runtime ID. This is useful for tracking runtime metrics across multiple processes.
```

Comment:
Are the descriptions for `DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED` and `DD_TRACE_EXPERIMENTAL_RUNTIME_ID_ENABLED`  flipped? The env var `DD_TRACE_EXPERIMENTAL_RUNTIME_ID_ENABLED` makes me think of `tagging runtime metrics with the current runtime ID`, but it's listed under `DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED`

Reply:
Ya they are